src/stores/Research/research.store.tsx

```diff
@@ -66,16 +66,30 @@ export class ResearchStore extends ModuleStore {
   public updateUploadStatus: IUpdateUploadStatus =
     getInitialUpdateUploadStatus()
 
+  public filterResearchesByCategory = (
+    collection: IResearch.ItemDB[] = [],
+    category: string,
+  ) => {
+    return category
+      ? collection.filter((obj) => {
+          return obj.researchCategory?.label === category
+        })
+      : collection
+  }
+
   constructor() {
     super(null as any, 'research')
     makeObservable(this)
     super.init()
 
     this.allDocs$.subscribe((docs: IResearch.ItemDB[]) => {
       logger.debug('docs', docs)
-      const sortedItems = docs.sort((a, b) =>
-        a._modified < b._modified ? 1 : -1,
-      )
+      const sortedItems = [...docs]
+        .filter((doc) => {
+          return !doc._deleted
+        })
+        .sort((a, b) => (a._modified < b._modified ? 1 : -1))
```

Comment:
We need to use the contentModifiedTimestamp now so that things like comments and useful votes don't change the listing position.

```suggestion
        .sort((a, b) => (a._contentModifiedTimestamp < b._contentModifiedTimestamp ? 1 : -1))
```